how-to-guides/data-versioning/scripts/Version_datasets_in_model_training_runs.py

```diff
@@ -0,0 +1,110 @@
+import neptune.new as neptune
+import pandas as pd
+from sklearn.ensemble import RandomForestClassifier
+
+TRAIN_DATASET_PATH = '../datasets/tables/train.csv'
+TEST_DATASET_PATH = '../datasets/tables/test.csv'
+
+
+def train_model(params, train_path, test_path):
+    train = pd.read_csv(train_path)
+    test = pd.read_csv(test_path)
+
+    FEATURE_COLUMNS = ['sepal.length', 'sepal.width', 'petal.length',
+                       'petal.width']
+    TARGET_COLUMN = ['variety']
+    X_train, y_train = train[FEATURE_COLUMNS], train[TARGET_COLUMN]
+    X_test, y_test = test[FEATURE_COLUMNS], test[TARGET_COLUMN]
+
+    rf = RandomForestClassifier(**params)
+    rf.fit(X_train, y_train)
+
+    score = rf.score(X_test, y_test)
+    return score
+
+
+#
+# Run model training and log dataset version, parameter and test score to Neptune
+#
+
+# Create a Neptune Run and start logging
+# run = neptune.init(project="common/quickstarts",
+#                    api_token="ANONYMOUS")
+run = neptune.init(project='common/data-versioning',
+                   api_token='ANONYMOUS')
+
+# Track dataset version
+run["datasets/train"].track_files(TRAIN_DATASET_PATH)
+run["datasets/test"].track_files(TEST_DATASET_PATH)
+
+# Log parameters
+PARAMS = {'n_estimators': 3,
+          'max_depth': 3,
+          'max_features': 1,
+          }
+run["parameters"] = PARAMS
+
+# Caclulate and log test score
+score = train_model(PARAMS, TRAIN_DATASET_PATH, TEST_DATASET_PATH)
+run["metrics/test_score"] = score
+
+# Get Neptune Run ID of the first, baseline model training run
+baseline_run_id = run['sys/id'].fetch()
+print(baseline_run_id)
+
+# Stop logging to the active Neptune Run
+run.stop()
+
+#
+# Change the model parameters
```

Comment:
Seems that you do this part below in line 94. I guess that it's sort of title for the remaining part, but when I read it for the first time I thought that there is some code missing :)